portfolio/src/main/webapp/index.html

```diff
@@ -66,8 +66,8 @@ <h2 style="background-color:skyblue">Comments</h2>
       <h3>Post a comment!</h3>
       <p id='login-or-logout-prompt'></p>
       <form id="comment-form" action="/comments" method="POST">
-        <p>Your name:</p>
-        <input type="text" id="name" name="name"><br>
+        <!-- <p>Your name:</p>
+        <input type="text" id="name" name="name"><br> -->
```

Comment:
Do you want to delete this?

Reply:
Yep, because their email will be displayed next to their comment instead so the name input never really gets used

Reply:
So please delete them from the code instead of commenting it

Reply:
Oh right, my bad - deleting now